src/components/templates/ConversationSpace/ConversationSpace.tsx

```diff
@@ -41,118 +23,40 @@ export interface ConversationSpaceProps {
-  useExperience(venue?.name);
+  const tables = venue?.config?.tables ?? CONVERSATION_TABLES;
 
-  const { userWithId } = useUser();
+  if (!userId) {
+    return <Loading />;
+  }
 
-  const generatedTables = venue?.config?.tables;
-
-  const infoIcon =
-    venue?.host?.icon ||
-    (PORTAL_INFO_ICON_MAPPING[venue.template] ?? DEFAULT_VENUE_LOGO);
-
-  const tables = generatedTables ?? CONVERSATION_TABLES;
   return (
     <>
-      <InformationLeftColumn iconNameOrPath={infoIcon}>
-        <InformationCard title="About the venue">
-          <p className="title-sidebar">{venue.name}</p>
-          <p className="short-description-sidebar">
-            {venue.config?.landingPageConfig.subtitle}
-          </p>
-          <div>
-            <RenderMarkdown
-              text={venue.config?.landingPageConfig.description}
-            />
-          </div>
-        </InformationCard>
-      </InformationLeftColumn>
-      <VenueWithOverlay venue={venue} containerClassNames="conversation-space">
-        {!seatedAtTable && parentVenueId && parentVenue && (
-          <BackButton variant="simple" space={parentVenue} />
-        )}
-        <div className={`scrollable-area ${seatedAtTable && "at-table"}`}>
-          {venue.description?.text && (
-            <div className="row">
-              <div className="col">
-                <div className="description">
-                  <RenderMarkdown text={venue.description?.text} />
-                </div>
-              </div>
-            </div>
-          )}
-          <div className="container-in-row">
-            <div className="video-wrapper">
-              {seatedAtTable && (
-                <TableHeader
-                  seatedAtTable={seatedAtTable}
-                  setSeatedAtTable={setSeatedAtTable}
-                  venueId={venue.id}
-                  venueName={venue.name}
-                  tables={tables}
-                  defaultTables={CONVERSATION_TABLES}
-                />
-              )}
-              {seatedAtTable && (
-                <div className="participants-container">
-                  <Room
-                    setSeatedAtTable={setSeatedAtTable}
-                    venueId={venue.id}
-                    roomName={`${venue.id}-${seatedAtTable}`}
-                  />
-                </div>
-              )}
-              {!seatedAtTable && (
-                <TablesControlBar
-                  containerClassName="ControlBar__container"
-                  onToggleAvailableTables={toggleTablesVisibility}
-                  showOnlyAvailableTables={showOnlyAvailableTables}
-                />
-              )}
+      <div className={styles.componentSpaceInfo}>
+        {venue.name && <h1>{venue.name}</h1>}
+      </div>
+
+      {venue.description?.text && (
+        <div className="row">
+          <div className="col">
+            <div className="description">
+              <RenderMarkdown text={venue.description?.text} />
```

Comment:
This has been repeated between here and the jazzbar. Can you extract it into a new component please?
Eventually it will look like this: https://www.figma.com/file/DltmqgBfdkgyX7z5Uv4r15/HiFi%3A-Visual-Redesign---Shared-w-Eng?node-id=658%3A9271

Reply:
absolutely! Will do. 🙃

Reply:
created `SpaceInfo` component: https://github.com/sparkletown/sparkle/pull/2910/commits/afe06cdd59c668832f14435166c1d3c97fba4064